### DISCUSSION TITLE
Show heads up/Pop up notification when app is in background?

### DISCUSSION THREAD

**Discussion by shivanshShekhar** [capawesome-team/capacitor-firebase#317] [Q&A] [answered]
I am able to show Push notifications both in case of background and foreground in notification panel. I am able to show heads up notification in case of foreground by creating a local notification channel of importance 5 and visibility 1.

Any way to do this when app is in background?

**Comment by robingenz** [comment, marked as answer]
Should be possible when https://github.com/capawesome-team/capacitor-firebase/issues/94 is implemented, which is part of the next release.

**Reply by shivanshShekhar** [reply]
Hi thanks for reply. Since there is currently no way to create channel. I was wondering if there is any way to show notification through Local Notifications when app is in background. Just like in foreground?

**Reply by robingenz** [reply]
> Since there is currently no way to create channel.

You should be able to create channels using the official Capacitor Push Notification plugin in the meantime: https://capacitorjs.com/docs/apis/push-notifications

> I was wondering if there is any way to show notification through Local Notifications when app is in background.

This question is about a plugin that is not part of this project. Please ask your question in https://github.com/ionic-team/capacitor-plugins.
